content/en/docs/components/trainer/operator-guides/podtemplate-overrides.md

```diff
@@ -31,6 +33,9 @@ podTemplateOverrides:
 
 The `PodTemplateOverrides` API supports various configuration options to customize Pod behavior.
 You can specify multiple overrides in the array, with later entries taking precedence over earlier ones.
+The overrides are applied in the following order: TrainingRuntime PodTemplate → PodTemplateOverrides 
```

Comment:
The priority is TrainJob (e.g. ML policy) > PodTemplateOverrides[n] > PodTemplateOverrides[n-1] > ... > PodTemplateOverrides[0] > TrainingRuntime where n is the number of PodTemplateOverrides